chatez/src/contexts/chatContext/index.tsx

```diff
@@ -6,6 +6,12 @@ interface ChatContextType {
   setCurrentFriend: (user: User | undefined) => void;
   loadMessages: boolean;
   setLoadMessages: (user: boolean) => void;
+  setAddedOrRemovedFriendStatus: (removeFriendStatus: boolean) => void;
+  addedOrRemovedFriendStatus: boolean;
```

Comment:
using boolean for a field named like this is confusing

Reply:
yh i know couldn't think of anything at that time